moveit_planners/chomp/chomp_motion_planner/src/chomp_planner.cpp

```diff
@@ -70,33 +78,89 @@ bool ChompPlanner::solve(const planning_scene::PlanningSceneConstPtr& planning_s
 
   if (req.goal_constraints.size() != 1)
   {
-    ROS_ERROR_NAMED("chomp_planner", "Expecting exactly one goal constraint, got: %zd", req.goal_constraints.size());
+    RCLCPP_ERROR(LOGGER, "Expecting exactly one goal constraint, got: %zd", req.goal_constraints.size());
     res.error_code_.val = moveit_msgs::msg::MoveItErrorCodes::INVALID_GOAL_CONSTRAINTS;
     return false;
   }
 
+  // Convert to joint space goal.
```

Comment:
I still need to review this in detail. While this seems to be a really useful addition, I'd prefer if you could split this into a separate PR since it's unrelated to the migration work you have done